### PR TITLE
Dashboard subscribing

### PR DESCRIPTION
**Pull request by paulauus**
Subscription works everywhere apart from uploading emails to SNS topics. Will debug together.

### CODE REVIEW

File: dashboard/pages/Games.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# pylint: disable=E0401
```
Comment (Leon-JS02) [inline dashboard/pages/Games.py:1]: Very descriptive

File: dashboard/functions.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def create_donut_chart():
         title=alt.TitleParams(
             text='Distribution of Game Releases by Weekday',
             anchor='end',
-            fontSize=17
+            fontSize=16
```
Comment (mehrdadhalali) [inline dashboard/functions.py:47]: Great change, powers of 2 are better than primes

File: dashboard/pages/Subscribe.py [hunk 3]
```diff
@@ -1,79 +1,98 @@
...
+    genre_topics_2 = {
+        "RPG": st.checkbox("RPG"),
+        "Strategy": st.checkbox("Strategy"),
+        "Sports": st.checkbox("Sports"),
+        "Racing": st.checkbox("Racing"),
+        "Multiplayer": st.checkbox("Multiplayer")
+    }
+
+# Get the selected genres (those that are checked)
+genre_topics.update(genre_topics_2)
```
Comment (Leon-JS02) [inline dashboard/pages/Subscribe.py:51]: Smart way to handle this

**Review by Leon-JS02 — APPROVED** [review]
Glad the code is working now. Future refactors could make it neater, though.

File: dashboard/pages/Subscribe.py [hunk 4]
```diff
@@ -1,79 +1,98 @@
...
 
 import re
 
 import streamlit as st
 
-# Regex for validating email
-email_pattern = r'^[\w\.-]+@[\w\.-]+\.\w+$'
+from pages.utils.subscribe_functions import add_subscriber_to_rds, create_subscriber_chart, subscribe_user_to_topics, remove_subscriber_from_rds, unsubscribe_user_from_all_topics
+
+EMAIL_PATTERN = r'^[\w\.-]+@[\w\.-]+\.\w+$'
```
Comment (mehrdadhalali) [inline dashboard/pages/Subscribe.py:9]: Love the fancy regex

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**